doc/source/toolkit/ui.rst

```diff
@@ -1,101 +1,97 @@
-============
-UI reference
-============
+==============
+Antenna wizard
+==============
 
-This section describes how to use the Antenna Toolkit Wizard, which requires an
-installed and licensed copy of AEDT. It assumes that you have already launched the
+This section describes how to use the Antenna wizard. It assumes that you have already launched the
```

Comment:
Name here is Antenna wizard but later Antenna Toolkit Wizard. Match what is shown in the wizard's UI!